src/platform/symbols.map

```diff
@@ -160,3 +160,11 @@ MIRPLATFORM_1.0 {
     mir::options::wayland_socket_name_opt*;
   };
 } MIRPLATFORM_0.27;
+
+# Uuuuh, 1.0 is not yet a thing.
+MIR_PLATFORM_0.31 {
```

Comment:
We've missed 0.31.

Although, as we're bumping ABI, can't we pull things together into MIR_PLATFORM_0.32?

Reply:
Hm. Have we bumped mirplatform ABI? I don't think this changes any symbols from `libmirplatform`. It does bump the graphics platform ABI, but that is separate from the mirplatform ABI.